projects/assets-library/assets/styles/_font.scss

```diff
@@ -341,3 +341,10 @@ $font-family: 'Work Sans', sans-serif;
 $mat-typography: mat.define-typography-config(
   $font-family: $font-family
 );
+
+@mixin line-clamp($maxLines: 1) {
```

Comment:
What's the most common usecase? Is it 1?

Reply:
Generally we truncate on single lines now.